editor/src/components/titlebar/title-bar.tsx

```diff
@@ -40,18 +40,24 @@ const ProjectTitle: React.FC<React.PropsWithChildren<ProjectTitleProps>> = ({ ch
 }
 
 const TitleBar = React.memo(() => {
-  const { dispatch, loginState, projectName, upstreamChanges } = useEditorState(
+  const { dispatch, loginState, projectName, upstreamChanges, currentBranch } = useEditorState(
     (store) => ({
       dispatch: store.dispatch,
       loginState: store.userState.loginState,
       projectName: store.editor.projectName,
       upstreamChanges: store.editor.githubData.upstreamChanges,
+      currentBranch: store.editor.githubSettings.branchName,
     }),
     'TitleBar',
   )
 
   const userPicture = useGetUserPicture()
 
+  const repoName = useEditorState(
```

Comment:
I went in and simplified this selector, previously it was a useEditorState + useMemo, but it can be done in a single useEditorState

Reply:
than you!